.github/workflows/ci.yml

```
@@ -33,9 +33,27 @@ jobs:
     runs-on: ubuntu-latest
     steps:
     - uses: actions/checkout@v4
+    - name: Set up date and time variable
+      run: echo "DATE_TIME=$(date '+%Y-%m-%d_%H-%M-%S')" >> $GITHUB_ENV 
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2086:info:1:50: Double quote to prevent globbing and word splitting

(shellcheck)